Cargo.toml

```diff
@@ -9,15 +9,15 @@ readme = "README.md"
 license-file = "LICENSE"
 keywords = ["lsm", "storage-engine", "embedded", "persistence"]
 repository = "https://github.com/helsing-ai/lsmlite-rs"
-rust-version = "1.64"
```

Comment:
Since we are bumping MSRV, shall we change `version` to `0.1.1`?

Reply:
Indeed. Done to 0.1.1-dev

Reply:
Thanks!